frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -54,42 +48,28 @@ import {
 } from "./EmbedFrame.styled";
 import { LogoBadge } from "./LogoBadge";
 
-type ParameterValues = Record<ParameterId, ParameterValueOrArray>;
+type ParameterValues = Record<ParameterId, ParameterValueOrArray | null>;
 
-interface OwnProps {
-  className?: string;
-  name?: string;
-  description?: string;
-  question?: Question;
-  dashboard?: Dashboard;
-  actionButtons?: JSX.Element[];
-  footerVariant?: FooterVariant;
-  parameters?: Parameter[];
-  parameterValues?: ParameterValues;
-  draftParameterValues?: ParameterValues;
-  hiddenParameterSlugs?: string;
-  enableParameterRequiredBehavior?: boolean;
-  setParameterValue?: (parameterId: ParameterId, value: any) => void;
+export type EmbedFrameBaseProps = Partial<{
```

Comment:
A bunch of these can already be null or undefined, so I just moved things to a `Partial` and set the appropriate values. This solves a bunch of type issues that `EmbedFrame` was exposing in `PublicDashboard`